skycultures/ticuna/info.ini

```diff
@@ -0,0 +1,5 @@
+[info]
+name = Ticuna
+author = Youssif Ghantous Filho
+descriptionSource = 
```

Comment:
descriptionSource is no valid key. Please see section 9.1 of the User Guide.

Reply:
Change made